Tabnine/src/main/java/com/tabnine/statusBar/TabnineStatusBarWidget.java

```diff
@@ -70,14 +73,21 @@ public TabnineStatusBarWidget(@NotNull Project project) {
             }
           }
         });
+
+    CompletionsStateNotifier.Companion.subscribe(isEnabled -> update());
```

Comment:
@yonip23 this is where I've handled the saas

Reply:
oh missed it 👍🏻